eng/build.py

```diff
@@ -0,0 +1,178 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+"""Build automation scripts"""
+
+import os
+import subprocess
+import sys
+from typing import List
+
+
+def _display_help():

```

Comment:
I'd just use the docstrings for displaying help... You can display docstrings in console by running `print(__doc__)`

Reply:
I had not thought about putting this information into a docstring. Let's consider that when making the next round of improvements, while working on build automation for Google Classroom.

Reply:
Oh, since it is not a class, it could be harder to display the entire docstring from a function... I think it is okay, from that perspective.